README.md

```diff
@@ -0,0 +1,31 @@
+# matrix-auth-plugin
+Matrix-based authorization strategies for Jenkins
+
+Sample pipeline (assumes matrix based security enabled on jenkins):
+
+    properties ([
+  		  [$class: 'AuthorizationMatrixProperty',
+    		    permissions : ['hudson.model.Item.Build:admin','hudson.model.Item.Build:user']
+    		]
+    ])
+    
+    node {
+        stage 'ekho'
+        sh 'echo bla'
+    }
+
+
+This will ensure admin and user have build permission on given project.
+
+Other permissions:
+* hudson.model.Item.Build
+* hudson.model.Item.Cancel
+* hudson.model.Item.Configure
+* hudson.model.Item.Read
```

Comment:
Probably also meaningless, since either the containing folder allows `READ`—in which case the branch project will inherit that too—or it does not—in which case you cannot even navigate to the branch project at all.